### DISCUSSION TITLE
Help with Debugging RPG Programs with DSPF

### DISCUSSION THREAD

**Discussion by antoine59910** [codefori/vscode-ibmi#1803] [Q&A]
Hello everyone,

I'm currently facing a challenge with debugging RPG programs that use Display Files (DSPF) on IBM i. While I'm comfortable with debugging programs that don't involve DSPFs, introducing a display file into the mix seems to complicate things for me. I don't know how to involve the DSPF... 
I receive an error message in QSYSOPR, which seems to be related to the file handling process.

Here's the error message I encounter:

Cause . . . . . : The RPG procedure KUTP30 in program PTFV230426/KUTP30 received message CPF4103 during an implicit OPEN operation on file KUTP30E. The actual file is KUTP30E.

Are there any particular settings or steps I should follow when setting up the debugger for programs that include DSPFs?

Any insights or guidance would be greatly appreciated.

Thank you in advance for your help!

**Comment by worksofliam** [comment]
Right now the IBM i Debugger for VS Code from IBM does not support interactive debugging. It will be possible when support for Service Entry Points has been released, though.
